src/query/select.rs

```diff
@@ -56,11 +56,12 @@ pub struct SelectStatement {
 }
 
 /// List of distinct keywords that can be used in select statement
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone)]
 pub enum SelectDistinct {
     All,
```

Comment:
I think we don't need `SelectDistinct::All`? I actually can't see it's being used and we don't need it anyways because `SELECT ALL` is the default behaviour.